src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final static int BOUND = 100;
+    private final String white = "WHITE";
+    private final static int radius = 10;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    private int getRandomNum() {
```

Comment:
```suggestion
    private int getRandomNumber() {
```

private methods should be below public as helper onew